NetKVM/Common/ParaNdis_TX.cpp

```diff
@@ -411,6 +412,7 @@ void CParaNdisTX::Send(PNET_BUFFER_LIST NBL)
         return;
     }
 
+    TraceNoPrefix(3, "%s:Q#%d NBL count = %d\n", __FUNCTION__, m_queueIndex, count);
```

Comment:
Don’t you want to print something in case flow in the "if (!m_StateMachine.RegisterOutstandingItems(count, &RejectionStatus))”? 
Anyway. looks OK

Reply:
Seems to me redundant. This just means we're in pausing process.